### PR TITLE
Send edge tracking request for suppressed in-app notifications

### PR DESCRIPTION
**Pull request by rymorale**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/MessagingFullscreenEventListener.java [hunk 1]
```diff
@@ -83,9 +86,41 @@ public void onDismiss(@NonNull final Presentable<InAppMessage> fullscreenMessage
...
+     * string will be returned for other {@code PresentationError} types.
+     *
+     * @param presentationError the {@link PresentationError} which occurred
+     * @return the error reason {@code String} if available, otherwise null
+     */
+    private static @Nullable String getErrorReason(
+            @NonNull final PresentationError presentationError) {
+        final Class presentationErrorClass = presentationError.getClass();
+        String errorReason = null;
+        if (presentationErrorClass.equals(SuppressedByAppDeveloper.class)) {
```
Comment (spoorthipujariadobe) [inline code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/MessagingFullscreenEventListener.java:116]: can we do `if(presentationError instanceOf SuppressedByAppDeveloper)` here?

**Review by spoorthipujariadobe — APPROVED** [review]
LGTM, small nit

File: code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/MessagingFullscreenEventListener.java (hunk 1 above)
Comment (rymorale) [reply code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/MessagingFullscreenEventListener.java:116]: good catch, updated this

**Review by sbenedicadb — APPROVED** [review]
looks good, just one constant name to update

**Pull request merged**